docs/src/pages/premium-themes/gmail/index.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import Checkbox from '@material-ui/core/Checkbox';
+import Tabs from '@material-ui/core/Tabs';
+import { makeStyles } from '@material-ui/styles';
+import Divider from '@material-ui/core/Divider';
+import Tooltip from '@material-ui/core/Tooltip';
+import IconButton from '@material-ui/core/IconButton';
+import Inbox from '@material-ui/icons/Inbox';
+import Social from '@material-ui/icons/Group';
+import LocalOffer from '@material-ui/icons/LocalOffer';
+import New from '@material-ui/icons/FiberNew';
+import Refresh from '@material-ui/icons/Refresh';
+import GmailLayout from './modules/layout';
+import GTab from './modules/components/tab';
```

Comment:
I think that we should keep using the `Tab` name as long as the component as the same structure than the Material-UI raw version.

The problem I always hear about wrapping is about knowing what version to use. I believe it's something that can be solved with tools. It could be a webpack alias, a babel alias or an eslint rule. I think that it's important to keep the same name to highlight the fact that it's only a style override.